docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -10,10 +10,39 @@ default:: `syn-blackbox-exporter`
 
 The namespace in which to deploy this component.
 
+== `helmValues`
+
+[horizontal]
+type:: dict
+default:: `{}`
+
+List of helm values to pass to the helm chart. For a list of supported values see: https://github.com/helm/charts/tree/master/stable/prometheus-blackbox-exporter
 
 == Example
 
 [source,yaml]
 ----
-namespace: example-namespace
+namespace: my-blackbox-exporter
+helmValues:
+  runAsUser: "1005499999"
```

Comment:
Side note: it's not gonna be fun having to configure this after the component has been deployed on OpenShift 4 (as you can't predict the UID range which your namespace will get assigned easily), might be worth to write a postprocessing filter which removes the fixed `runAsUser` config in the deployment for OpenShift 4 to let OpenShift assign a random UID to the resulting pods.

Reply:
Yeah, we figured out that you can just set it to `null` here to avoid that

Reply:
> Yeah, we figured out that you can just set it to null here to avoid that

Seems like something that should be documented somewhere in the component docs then

Reply:
I wonder if I should set it to null as default in the component. What do you think?

Reply:
Generally, on non-OCP you'll want the `runAsUser` field to be set to avoid having the image run as root, so I don't know whether setting it to `null` in the component defaults is the best option.

Reply:
Ok. Will not set it by default then in the component and will leave it as is. Updated the documentation though